google/cloud/aiplatform/models.py

```diff
@@ -2062,6 +2106,30 @@ def predict(self, instances: List, parameters: Optional[Dict] = None) -> Predict
             deployed_model_id=self._gca_resource.deployed_models[0].id,
         )
 
+    def raw_predict(
+        self, body: bytes = None, headers: Dict[str, str] = None
+    ) -> requests.models.Response:
+        """Make a prediction request using arbitrary headers.
+        This method must be called within the network the PrivateEndpoint is peered to.
+        The function call will fail otherwise. To check, use `PrivateEndpoint.network`.
```

Comment:
Do we know the specific error that would be raised?  If so perhaps add a `Raises:` section to the docstring: https://google.github.io/styleguide/pyguide.html#doc-function-raises

Reply:
I meant specifically the part “the function call will fail” in the docstring.

Reply:
Got it. I updated the doc strings in `PrivateEndpoint.predict` and `PrivateEndpoint.raw_predict` to contain information about the error code. Most common error code will be 404 with a message saying "request not found".